### DISCUSSION TITLE
需要朋友动态

### DISCUSSION THREAD

**Discussion by Bian-Sh** [community/community#7685] [Mobile] [answered]
我需要看到每日关注者的动态，通过他们发现更多先进的项目
同时还可以把本应用社交功能强化起来
![image](https://user-images.githubusercontent.com/24520547/141667005-ce82632f-9af7-419b-8181-a248c8894f4b.png)

**Comment by brianlovin** [comment, marked as answer]
Thanks for the note! We plan to add more social/discover/explore features in the future to make it easier to keep up with the people and projects you follow on GitHub.

**Reply by Bian-Sh** [reply]
It is really a good news, cheers!!!
